debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/SourceInfo.java

```diff
@@ -132,8 +130,6 @@
                                                      .field(BINLOG_ROW_IN_EVENT_OFFSET_KEY, Schema.INT32_SCHEMA)
                                                      .field(SNAPSHOT_KEY, SchemaBuilder.bool().optional().defaultValue(false).build())
                                                      .field(THREAD_KEY, Schema.OPTIONAL_INT64_SCHEMA)
-                                                     .field(DB_NAME_KEY, Schema.OPTIONAL_STRING_SCHEMA)
```

Comment:
Is there any impact in terms of compatibility (when using the registry) due to the changed order? Seems it'd be safer to leave it as is?

Reply:
@gunnarmorling Unfortunately you are right, for Avro it breaks backward compatibility. Can't we afford to do it in minor version change and thus postpone this to 0.10?